src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
```

Comment:
Pay attention to CSS properties order. Check yourself everywhere
https://9elements.com/css-rule-order/